packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -74,8 +74,8 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     this._ariaDescriptionId = getId();
     let menuProps = null;
     if (props.persistMenu && props.menuProps) {
-      menuProps = props.menuProps;
-      menuProps.hidden = true;
+      // Clone props so we don't mutate them.
```

Comment:
can we add a unit test :)

Reply:
@cliffkoh  done

Reply:
@khmakoto, @joschect has added a `test` for this. Could you please take a look?